onadata/apps/api/management/commands/delete_users.py

```diff
@@ -0,0 +1,126 @@
+"""
+Delete users management command.
+"""
+import sys
+from django.contrib.auth.models import User
+from onadata.apps.logger.models import XForm
+from onadata.apps.logger.models import Instance
+from onadata.apps.logger.models import Project
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+
+
+def get_user_object_stats(
+        username, email, user_input=None):  # pylint: disable=R0201
+    """
+    Get User information.
+    """
+    # Get the number of projects for this user
+    user_projects = Project.objects.filter(
+        created_by__username=username).count()
+    # Get the number of forms
+    user_forms = XForm.objects.filter(
+        user__username=username).count()
+    # Get the number of submissions
+    user_sumbissions = Instance.objects.filter(
+        user__username=username).count()
+    result = {
+        'projects': user_projects,
+        'forms': user_forms,
+        'submissions': user_sumbissions
+    }
+
+    user_response = input(
+        "User account '{}' has {} projects, "
+        "{} forms and {} submissions. "
+        "Do you wish to continue "
+        "deleting this account?".format(
+            username,
+            result['projects'],
+            result['forms'],
+            result['submissions']
+            ))
+
+    return user_response
+
+
+def inactivate_user(username, email):
+    """
+    Soft deletes the user termporarily.
+    """
+    try:
+        user = User.objects.get(username=username, email=email)
+        # set inactive status on user account
+        user.is_active = False
+        # append a timestamped suffix to the username
+        # to make the initial username available
+        soft_deletion_time = timezone.now()
+        deletion_suffix = soft_deletion_time.strftime('-deleted-at-%s')
```

Comment:
I think we do `deletion_suffix = timezone.now().strftime('-deleted-at-%s')`, this removes the need for `soft_deletion_time`

Reply:
Sure, changed that